File: core/src/main/java/net/minestom/server/utils/math/LongRange.java
package net.minestom.server.utils.math;

public class LongRange extends Range<Long> {

    public LongRange(Long minimum, Long maximum) {
        super(minimum, maximum);
    }

    public LongRange(Long value) {
        super(value);
    }

    /**
     * {@inheritDoc}
     */
    @Override
    public boolean isInRange(Long value) {
        return value >= this.getMinimum() && value <= this.getMaximum();
    }
}

File: src/main/java/net/minestom/server/utils/math/LongRange.kt
package net.minestom.server.utils.math

class LongRange : Range<Long> {
    constructor(minimum: Long, maximum: Long) : super(minimum, maximum) {}
    constructor(value: Long) : super(value) {}

    /**
     * {@inheritDoc}
     */
    override fun isInRange(value: Long): Boolean {
        return value >= minimum && value <= maximum
    }
}